boot/freeldr/freeldr/lib/peloader.c

```diff
@@ -473,6 +560,13 @@ PeLdrpScanImportAddressTable(
 
 /* FUNCTIONS *****************************************************************/
 
+PVOID
+PeLdrInitSecurityCookie(
```

Comment:
Is this supposed to be some kind of wrapper function? This doesn't seem like a good solution to me.